File: Contest/LeetCodeWeeklyContest/WeeklyContest248/3_fail.py
class Solution:
    def countGoodNumbers(self, n: int) -> int:
        """
        even = 0, 2, 4, 6, 8
        prime = 2, 3, 5, 7
        """
        MOD = 10 ** 9 + 7
        ans = 1
        for i in range(n):
            if i % 2 == 0:
                ans *= 5
            else:
                ans *= 4
            ans %= MOD
        return ans

# TLE
# 806166225460393

File: Contest/LeetCodeWeeklyContest/WeeklyContest248/3_fail.cpp
#include <math.h>
class Solution
{
public:
    int countGoodNumbers(long long n)
    {
        long long MOD = pow(10, 9) + 7;
        long long ans = 1;
        for (long long i = 0; i < n; i++)
        {
            if (i % 2 == 0)
            {
                ans *= 5;
            }
            else
            {
                ans *= 4;
            }
            ans %= MOD;
        }

        return ans;
    }
};

// TLE
// 806166225460393
